README.md

```diff
@@ -1,113 +1,100 @@
+Alongside streaming or viewing streams, users can also interact with each other through a live chat in each stream, and communicate with the stream host.
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Authors:
 
-## Available Scripts
+Emilio Alvarez Veronesi, Jeffery Huo, Jeffery Swarts, Joshua Velasquez, Yue Zhang
 
-In the project directory, you can run:
+## Website Link:
 
-### `npm start`
+[https://twijj-271803.web.app/](https://twijj-271803.web.app/)
 
-Runs the app in the development mode.<br>
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.<br>
-You will also see any lint errors in the console.
-
-### `npm test`
-
-Launches the test runner in the interactive watch mode.<br>
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `npm run build`
-
-Builds the app for production to the `build` folder.<br>
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.<br>
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Associated Github Repositories
 
-### `npm run eject`
+- Client Side -> [https://github.com/josh-velasquez/TwijjChatServer](https://github.com/josh-velasquez/Twijj)
+- Chat Server Side -> [https://github.com/josh-velasquez/TwijjChatServer](https://github.com/josh-velasquez/TwijjChatServer)
+- Stream Server Side -> [https://github.com/josh-velasquez/TwijjStreamServer](https://github.com/josh-velasquez/TwijjStreamServer)
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+## Tools and Applications Used
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+- Database -> Firebase
+- Server -> AWS (Chat and Stream)
+- Hosting -> Google platform/Firebase
 
-Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+## Web Browser Setup
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+### Allowing Insecure Content
 
-## Learn More
+Once the webapp is launched, navigate to your browser settings and then to site settings and allow for insecure content.
```

Comment:
You can also click the lock icon on the address bar: 
![image](https://user-images.githubusercontent.com/25234283/79012195-f5bb0380-7b22-11ea-89f9-54559402c22c.png)

Also, I think this section is true for the chat as well.